src/apps/loan-list/materials/utils/warning-bar.tsx

```diff
@@ -48,11 +52,15 @@ const WarningBar: FC<WarningBarProps> = ({
           >
             {rightText}
           </p>
+          <span className="hide-visually" id={labelId}>
+            {rightButtonAriaLabelText}
+          </span>
           {rightLink && (
             <Link
               dataCy="warning-bar-right-link"
               href={rightLink}
-              className="btn-primary btn-filled btn-small arrow__hover--right-small"
+              className="btn-primary btn-filled btn-small arrow__hover--right-small hide-linkstyle"
```

Comment:
Can't we override the link styling in the design system?
We know we have multiple places where a link button is rendered with an underline, and adding this utility class everywhere is not the solution. There should not be a case where we want an underline.

Reply:
https://github.com/danskernesdigitalebibliotek/dpl-design-system/pull/371